server/middlewares/validate.js

```diff
@@ -24,9 +24,14 @@ const titleCheck = body("title")
 const imageUrlCheck = body("imageUrl")
   .isURL()
   .withMessage("imageUrl must be a URL.");
+
+const isPrivateCheck = body("isPrivate")
+  .isBoolean()
+  .withMessage("isPrivate must be a boolean");
+
 const titleOrImageUrlCheck = oneOf(
```

Comment:
would probably be better if this validation were now titled `listBodyCheck` or something similar.

Reply:
I'll keep the status quo for now as I fear to break the whole thing hahaha